ByBit.Net/Objects/Models/Spot/v3/BybitBorrowOrderV3.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Newtonsoft.Json;
+
+namespace Bybit.Net.Objects.Models.Spot.v3
+{
+    /// <summary>
+    /// Borrow order place responce
+    /// </summary>
+    public class BybitBorrowOrderV3
+    {
+        /// <summary>
+        /// Borrow order transaction id
+        /// </summary>
+        [JsonProperty("transactId")]
+        public decimal TransactionId { get; set; }
```

Comment:
Is the transation id a decimal? I would guess a long or string, but not a decimal value

Reply:
What value type from V1 do  you mean?

Reply:
In Spot V1 it returns a decimal value (which is a transactional ID. In V3 they return a JSON class instead.

Reply:
Yes, I see your point. I've resolved it in my previous commit, please check it.